dspace-api/src/main/java/org/dspace/authority/orcid/Orcidv2.java

```diff
@@ -0,0 +1,145 @@
+package org.dspace.authority.orcid;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.dspace.authority.AuthorityValue;
+import org.dspace.authority.SolrAuthorityInterface;
+import org.dspace.authority.orcid.xml.XMLtoBio;
+import org.dspace.authority.orcid.xml.XMLtoWork;
+import org.dspace.authority.rest.RESTConnector;
+import org.json.JSONObject;
+import org.orcid.jaxb.model.record_v2.Person;
+import org.orcid.jaxb.model.record_v2.Work;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by jonas - jonas@atmire.com on 13/04/2018.
+ */
+public class Orcidv2 implements SolrAuthorityInterface {
```

Comment:
Could we get some basic code comments in this file?  I really don't like merging code that has zero code comments.  Ideally, all `public` classes and methods should have a basic comment describing them.

Reply:
We will include some basic code comments. They were not present in the ORCID v1 API, and therefore not included here yet